impeller/renderer/backend/vulkan/blit_command_vk.cc

```diff
@@ -0,0 +1,178 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "impeller/renderer/backend/vulkan/blit_command_vk.h"
+
+#include "impeller/renderer/backend/vulkan/commands_vk.h"
+#include "impeller/renderer/backend/vulkan/texture_vk.h"
+
+namespace impeller {
+
+BlitEncodeVK::~BlitEncodeVK() = default;
+
+// BEGIN: BlitCopyTextureToTextureCommandVK
```

Comment:
Not sure what these BEGIN, END blocks mean. Can we remove these?

Reply:
Oh, I see these are just code separators. We've [used this pattern in other TUs](https://github.com/flutter/engine/blob/89c8a1393d4b4ec567e8db9e8214bb6068a1b561/impeller/scene/importer/vertices_builder.h#L97). Can we do the same here please? Just a nit.